resources/keboola.ex-mailchimp/templates/api.json

```diff
@@ -17,5 +17,8 @@
   "name": "mailchimp",
   "authentication": {
     "type": "basic"
+  },
+  "retryConfig": {
+    "maxRetries": 5
```

Comment:
@ondrajodas 
Myslis, ze to pomoze? Ja si myslim, ze nie.

Default retry je 10x: https://github.com/keboola/juicer/blob/05b7f8d7711635552306513c6e8811aa8465ed88/Client/RestClient.php#L256

(naschval linkujem stary kod, nova verzia este nie je nasadena, mam na to task, ... ale aj v novom je tam 10x)

Reply:
Este na doplnenie

Generic extractor `2.3.2` pouziva juicer `3.2.2`
https://github.com/keboola/generic-extractor/blob/04041cfb4d4136e4978c7a4feadf5bbc4466b928/composer.lock#L617

A aj v juicer `3.2.2` je tam 10x:
https://github.com/keboola/juicer/blob/64fd9b31b9965c11a86c982c73d5328035f2e294/Client/RestClient.php#L218

Reply:
Netvrdim, ze to retry v tej starej verzii aj fungovalo, to bohuzial neviem povedat.

Reply:
V jobe sa to loguje asi iba pri debug-u (alebo vzdy? neviem ... ): https://github.com/keboola/juicer/blob/64fd9b31b9965c11a86c982c73d5328035f2e294/Client/RestClient.php#L246

Takze skor by som namiesto `5x` dal `15x` a po case to vyhodnotil.

Reply:
nj, toho jsem si nevšiml že je default na 10... nastavím tam teda 15 + jsem nastavil pro mailchimp poslední verzi genericu 

to jestli retry fungovalo na té starší verzi taky bohužel nevím

Reply:
ok

Reply:
> V jobe sa to loguje asi iba pri debug-u (alebo vzdy? neviem ... )

jj to je jen při debugu
https://github.com/keboola/juicer/blob/64fd9b31b9965c11a86c982c73d5328035f2e294/Client/RestClient.php#L246